server/src/modules/customer/customer.validator.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from "zod"
+
+export const createCustomerValidator = z.object({
+  name: z.string().min(2, "Enter minimum 2 characters"),
+  email: z.string(),
```

Comment:
We can update the validation of email with `z.string().email()`

Reply:
done